### PR TITLE
kjv: add page

### PR DESCRIPTION
**Pull request by FukurouMakoto**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/kjv.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`kjv John 3:16`
+
+- Open a range of verses in a book chapter:
+
+`kjv Proverbs 3:1-6`
+
+- Get a range of chapters and/or verses in a book:
+
+`kjv Matthew 1:7-2-6`
```
Comment (navarroaxel) [inline pages/linux/kjv.md:28]: This command has a typo:
```suggestion
`kjv {{Matthew}} {{1}}:{{7}}-{{2}}:{{6}}`
```

File: pages/linux/kjv.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Open individual verse(s) of a specific chapter of a book:
+
+`kjv John 3:16`
+
+- Open a range of verses in a book chapter:
+
+`kjv Proverbs 3:1-6`
+
+- Get a range of chapters and/or verses in a book:
```
Comment (navarroaxel) [inline pages/linux/kjv.md:26]: Your command example is about verses, not chapters. So, this example description is confused. Maybe this is better:
```suggestion
- Display a specific range of verses of a book from different chapters:
```

File: pages/linux/kjv.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`kjv Matthew 1:7-2-6`
+
+- Search all verses that match a pattern:
```
Comment (navarroaxel) [inline pages/linux/kjv.md:30]:
```suggestion
- Display all verses that match a pattern:
```

File: pages/linux/kjv.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`kjv Exodus/Plagues`
```
Comment (navarroaxel) [inline pages/linux/kjv.md:32]: Based on the example description this command is not right:
```suggestion
`kjv /{{Plagues}}`
```

File: pages/linux/kjv.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Search all verses in a book that match a pattern:
+
+`kjv 1Jn/antichrist`
```
Comment (navarroaxel) [inline pages/linux/kjv.md:36]:
```suggestion
`kjv {{1Jn}}/{{antichrist}}`
```

File: pages/linux/kjv.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Get a range of chapters and/or verses in a book:
+
+`kjv Matthew 1:7-2-6`
+
+- Search all verses that match a pattern:
+
+`kjv Exodus/Plagues`
+
+- Search all verses in a book that match a pattern:
```
Comment (navarroaxel) [inline pages/linux/kjv.md:34]:
```suggestion
- Display all verses that match a pattern in a specific book:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @FukurouMakoto, welcome and thanks for your submission. I left some comments.

File: pages/linux/kjv.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`kjv -l`
+
+- Open an individual book:
+
+`kjv Genesis`
+
+- Open an individual chapter of a book:
+
+`kjv Genesis 2`
```
[the remark this answers is not in the document]
Comment (FukurouMakoto) [reply pages/linux/kjv.md:16]: Thank you for taking your time to review my pull request. 
In the future, should I follow this example when typing out arguments/parameters for this project?

File: pages/linux/kjv.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Open individual verse(s) of a specific chapter of a book:
```
[the remark this answers is not in the document]
Comment (FukurouMakoto) [reply pages/linux/kjv.md:18]: I think that's fair. 
My concern was for people who possibly have never read the Bible before and thus would not know this, but after some consideration it seems unlikely such a person would make a Linux terminal their first place to do so.

**Comment by FukurouMakoto** [comment]
@navarroaxel Thanks for your help and for reviewing the document. The changes have all been made.

**Review by navarroaxel — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**